pkg/platform/model/vcs.go

```diff
@@ -424,8 +424,15 @@ func AddChangeset(parentCommitID strfmt.UUID, commitMessage string, changeset Ch
 
 	res, err := mono.New().VersionControl.AddCommit(params, authentication.ClientAuth())
 	if err != nil {
-		multilog.Error("AddCommit Error: %s", err.Error())
-		return nil, locale.WrapError(err, "err_add_commit", "", api.ErrorMessageFromPayload(err))
+		switch err.(type) {
+		case *version_control.AddCommitBadRequest:
+		case *version_control.AddCommitConflict:
+		case *version_control.AddCommitForbidden:
```

Comment:
Should these be ignored?

Reply:
I feel like these are also input errors. The BadRequest is if the user submits a bad request (not exactly sure what this entails). Conflict is what was reported to Rollbar -- the user tried to remove a package that did not exist. And Forbidden is an authentication error.

Reply:
If I'm understanding correctly, the case should then be `case badReq, conflict, forbidden:` since the current form will return the payload without error.

Reply:
`fallthrough` may also work, but in all my years writing Go I can't think of a time I preferred or even ended up using it.

Reply:
Thanks for the tip. I come from C/C++ land where fallthrough is implicit without a `break`. I was expecting the same in Go, but I see that is not the case.